test/e2e/scheduling/priorities.go

```diff
@@ -445,15 +453,16 @@ func addRandomTaitToNode(cs clientset.Interface, nodeName string) *v1.Taint {
 	return &testTaint
 }
 
-// updateMemoryOfNode updates the memory of given node with the given value
-func updateMemoryOfNode(c clientset.Interface, nodeName string, memory int64) error {
+// updateNodeAllocatable updates the allocatable values of given node with the given values.
+func updateNodeAllocatable(c clientset.Interface, nodeName string, memory int64, cpu int64) error {
```

Comment:
```suggestion
func updateNodeAllocatable(c clientset.Interface, nodeName string, memory, cpu int64) error {
```

Reply:
Style issue again. I don't mind changing it but AFAIK there is no standard preference I think.